cypress/integration/tests/controls/businessservices/create.test.ts

```diff
@@ -37,16 +37,16 @@ describe("Business service validations", () => {
         cy.get(commonView.cancelButton).should("not.be.disabled");
 
         // Name constraints
-        inputText(businessServiceNameInput, "te");
-        cy.get(commonView.nameHelper).should("contain", nameLeastChars);
-        inputText(businessServiceNameInput, data.getLongString());
-        cy.get(commonView.nameHelper).should("contain", nameMaxChars);
+        inputText(businessServiceNameInput, data.getRandomWord(2));
```

Comment:
are you sure those two words will have less than 3 chars?

Reply:
Yes. It generates a word of length 2 characters. This all changes have been run, and then only used.